Packs/RandomImages_VideosAndAudio/Scripts/RandomPhotoNasa/RandomPhotoNasa.yml

```diff
@@ -0,0 +1,21 @@
+args:
+- description: Search query
```

Comment:
Please consider adding predefined values, an example to the description and default value of `nebula`

Reply:
Done